### ISSUE TITLE
TOC of book is present but each spine item is empty

### ISSUE THREAD

**Issue by leonardr** [NYPL-Simplified/android#319]
I can use Overdrive Read to read Overdrive book E05B4505-0813-4337-9C91-ADE3FB557DFD, "A Fortress In Shadow" by Glen Cook. I can also fulfill this book in SimplyE. I can open the EPUB, and I can see the table of contents. But each spine item in the table of contents renders as a single blank page.

This is probably a cross-platform Readium issue.

This is different from https://github.com/NYPL-Simplified/Simplified-iOS/issues/433, where the entire book renders as blank and the TOC is empty.

**Comment by rkwright** [comment]
@leonardr  I am skeptical that this is a purely Readium issue as I have never seen this before.  But we would be happy to look at it.  Can you provide a sample of the book?  Clearly, we don't need the whole book, only a couple of chapters and the TOC.  
I understand that the book has DRM and is proprietary to OverDrive, but my experience at Adobe (where we developed RMSDK and ACS) is that publishers/providers were happy to provide subsets of books to resolve such problems as long as we promised to act swiftly, never distribute the sample and delete it when the problem was resolved.

**Comment by leonardr** [comment]
That's true, it could equally be a problem with the Adobe DRM layer. However, a similar error was reported on another book by the same author, which makes me think it might be a problem with the content.

You can get a DRM-free excerpt of the book from http://excerpts.cdn.overdrive.com/FormatType-410/1486-1/E05/B45/05/FortressInShadow9781597803779.epub. Given the large number of variables here, I don't know how much we can conclude if Readium renders the excerpt successfully, but a failure would be informative.

**Comment by leonardr** [comment]
This problem has also been reported on Android for THE BOOK OF NIGHT WITH MOON (Diane Duane), THIS IS REAL AND YOU ARE COMPLETELY UNPREPARED (Alan Lew), and TO VISIT THE QUEEN (Diane Duane). I have confirmed it with the Alan Lew book. 

Interestingly enough, there is one spine item of the Alan Lew that appears to render properly: the "Author's Note", which is probably only one page in the original.
